internal/plugins/ansible/v1/init.go

```diff
@@ -176,11 +176,11 @@ func addInitCustomizations(projectName string) error {
 	// todo: remove it when we solve the issue operator-framework/operator-sdk#3573
 	const resourcesLimitsFragment = `  resources:
           limits:
-            cpu: 200m
-            memory: 100Mi
+            cpu: 500m
+            memory: 128Mi
           requests:
-            cpu: 100m
-            memory: 20Mi
+            cpu: 10m
+            memory: 64Mi
```

Comment:
@camilamacedo86 Where did you get these values? @asmacdo @fabianvf any concerns with these values?

Reply:
These values are the default ones in Golang. 

Before the values were lower and we had the need to overwrite the Golang ones with new ones increased specifically for the helm to let run the sample. Now, we do not need it anymore and we can simplify things. See: https://github.com/operator-framework/operator-sdk/pull/5330#discussion_r738565284

Also, it is important to highlight that the default values ought to be enough only for a getting start (sample). Users ought to change them according to the needs of their solutions. See that we scaffold the TODO now as well. 

However, we must scaffold them by default. The big issue is when the operator does not have them set and we indeed we need to make it mandatory for the request. https://github.com/operator-framework/api/issues/172

I have no concern with these values and indeed we can check that our e2e tests are working well.

Reply:
I don't see a problem, lgtm.

Reply:
@camilamacedo86 actually it looks like CI time for ansible has increased significantly between the first iteration of your PR 4 days ago: https://github.com/operator-framework/operator-sdk/actions/runs/1454346342 and the most recent one: https://github.com/operator-framework/operator-sdk/actions/runs/1454351107

( 19m 48s  vs  1h 25m 53s )